src/pages/Search/index.js

```diff
@@ -1,11 +1,27 @@
 import React from 'react';
+import dayjs from 'dayjs';
+import axios from 'axios';
 import SearchContainer from './styles';
 import Form from './components/Form';
+import config from '../../constants/config';
 
-const SearchPage = () => (
-  <SearchContainer>
-    <Form />
-  </SearchContainer>
-);
+const SearchPage = () => {
+  const loadData = async (subreddit) => {
```

Comment:
This can be moved outside of SearchPage (still inside this file though). Then it's clear that it's not depending on any outside parameters. It's then clearly marked as a pure function